src/pyfx/view/common/frame.py

```diff
@@ -278,12 +278,12 @@ def get_cursor_coords(self, size):
         return x, y + row_adjust
 
     @overrides
-    def selectable(self):
+    def selectable(self) -> bool:
         """Frame is always selectable."""
         return True
 
     @overrides
-    def sizing(self):
+    def sizing(self) -> frozenset[urwid.Sizing]:
```

Comment:
Looks like this is not supported in Py38, can you also remove it? https://stackoverflow.com/questions/75202610/typeerror-type-object-is-not-subscriptable-python

Reply:
Or we can remove support for Py38, since it is an old version.

Reply:
What do you think about the solution suggested in the SO you linked? i.e. `def sizing(self) -> typing.FrozenSet[urwid.Sizing]:`?

> can you also remove it?

I could remove the `@overrides`, if that's what you mean?

> remove support for Py38

That seems avoidable.